app/components/table.js

```diff
@@ -39,8 +39,8 @@ const StyledTable = styled.table`
   }
 `
 
-const Table = ({ dats, shareDat, onDeleteDat }) => (
-  <main>
+const Table = ({ dats, showInspect, shareDat, onDeleteDat, inspectDat }) => (
+  <main style={{ display: showInspect ? 'none' : 'block' }}>
```

Comment:
I think the cleanest solution here is not to `display:none` but not render the component at all. What about renaming the `showInspect` property to simply `show` and returning an empty `<Fragment />` in that case?

So it would be `<Table show={false}>` if inspect is to be shown.